functorch/test/test_vmap.py

```diff
@@ -3330,6 +3333,9 @@ def test_vmap_exhaustive(self, device, dtype, op):
         xfail('any'),
         xfail('count_nonzero'),
         xfail('nanmean'),
+        xfail('nn.functional.conv1d'),  # AssertionError: Exception not raised on ErrorInput
```

Comment:
cc: @zou3519

Reply:
Can you file an issue to track this?

Reply:
Sure, @zou3519! Thanks for your input. Raised an issue: #85656.